core/src/main/java/org/infinispan/configuration/cache/PersistenceConfigurationBuilder.java

```diff
@@ -192,13 +209,30 @@ public ClusterLoaderConfigurationBuilder addClusterLoader() {
 
    /**
     * Adds a single file cache store.
+    * @deprecated since 13.0. To be removed in 14.0 has been replaced by {@link #addSoftIndexFileStore()}
```

Comment:
Isn't this too soon for removal downstream?

Reply:
The rule for downstream is has to span a .0 to .0 before it can be released. Since it would be in 13.0 that means it can be removed in 14.0. At least that is my understanding.

Reply:
My understanding was that the code/feature had to be deprecatef for at least two downstream minors before removal. So that would be 16.0 when it can be removed. As an example see [ISPN-11691](https://issues.redhat.com/browse/ISPN-11691) which defines code to be removed in 14.0. Alll of the linked Jiras contain code that was deprecated in 11.0.

\\cc @tristantarrant In case this has changed or I have missunderstood.